### ISSUE TITLE
404 when linking a file from resources

### ISSUE THREAD

**Issue by ohmtech-rdi** [electro-smith/DaisyExamples#281]
Some links in the documentation are broken.

For example in the [Drum](https://github.com/electro-smith/DaisyExamples/tree/master/seed/DSP/Drum) sample code:

The breadboard illustration links to 
```
https://raw.githubusercontent.com/electro-smith/DaisyExamples/master/seed/Drum/resources/Drum_bb.png
```

But it seems that GitHub has deprecated those kind of links, so this will return a 404.

It should be probably replaced with:
```
https://github.com/electro-smith/DaisyExamples/blob/master/seed/DSP/Drum/resources/Drum_bb.png
```

(Originally reported by K. Shepherd on Daisy Slack)

**Comment by stephenhensley** [comment]
this has been resolved.

Thanks for making the issue!

**Comment by ohmtech-rdi** [comment]
- For reference, fixed in #282
